src/order/hom/complete_lattice.lean

```diff
@@ -44,13 +44,14 @@ structure Inf_hom (α β : Type*) [has_Inf α] [has_Inf β] :=
 (map_Inf' (s : set α) : to_fun (Inf s) = Inf (to_fun '' s))
 
 /-- The type of frame homomorphisms from `α` to `β`. They preserve `⊓` and `⨆`. -/
```

Comment:
If this PR really is a definition change, either this docstring was wrong before, is wrong now, or was too vague in both cases.

Reply:
It was correct about the code, but not the mathematical definition. Since the code was updated:
```suggestion
/-- The type of frame homomorphisms from `α` to `β`. They preserve finite meets and arbitrary joins. -/
```